tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/VersionMismatchTests.cs

```diff
@@ -0,0 +1,148 @@
+// <copyright file="VersionMismatchTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using Datadog.Trace.Ci.CiEnvironment;
+using Datadog.Trace.Ci.Ipc;
+using Datadog.Trace.Ci.Ipc.Messages;
+using Datadog.Trace.Ci.Tags;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.ExtensionMethods;
+using Datadog.Trace.TestHelpers;
+using Datadog.Trace.TestHelpers.Ci;
+using Datadog.Trace.Util;
+using Datadog.Trace.Vendors.Newtonsoft.Json;
+using FluentAssertions;
+using VerifyXunit;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.CI;
+
+[UsesVerify]
```

Comment:
I don't think you are using Verify here
```suggestion
```

Reply:
done in [f337ea8](https://github.com/DataDog/dd-trace-dotnet/pull/5578/commits/f337ea8c318021939998571bf8f682e8012b8257)